<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FillDialog</class>
 <widget class="QDialog" name="FillDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="methodLabel">
       <property name="text">
        <string>Method:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="boundaryLabel">
       <property name="text">
        <string>Boundary:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="neighborhoodLabel">
       <property name="text">
        <string>Neighborhood:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ColorEdit" name="boundaryColorEdit" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="methodGroupBox">
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="methodHorizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="floodRadioButton">
          <property name="text">
           <string>Flood fill</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="boundaryRadioButton">
          <property name="text">
           <string>Boundary fill</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QGroupBox" name="neighborhoodGroupBox">
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="typeHorizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="fourConnectedRadioButton">
          <property name="text">
           <string>&amp;4-connected</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="eightConnectedRadioButton">
          <property name="text">
           <string>&amp;8-connected</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorEdit</class>
   <extends>QWidget</extends>
   <header>widgets/coloredit.h</header>
   <container>1</container>
   <slots>
    <signal>colorEdited(QColor)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FillDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FillDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boundaryRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>boundaryColorEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
